src/backend/access/external/pxffilters.c

```diff
@@ -330,7 +329,27 @@ pxf_free_filter(PxfFilterDesc* filter)
  *
  * Yields the following serialized string:
  *
- * a0c1o2a0c5o1o7a2c"third"o5o7
+ * a0c23s1d1o2a1c23s1d5o1a2c25s5dthirdo5l0l0
```

Comment:
Update above comment Each filter is serialized as follows:
<attcode><attnum><constcode><constval><constsizecode><constsize><constdata><constvalue><opercode><opernum>